master/docs/manual/cfg-workers.rst

```diff
@@ -47,14 +50,22 @@ For example::
                       properties={ 'os':'solaris' }),
     ]
 
+Limiting Concurrency
+++++++++++++++++++++
+
 .. index:: Workers; limiting concurrency
 
-The :class:`Worker` constructor can also take an optional ``max_builds`` parameter to limit the number of builds that it will execute simultaneously::
+The :class:`Worker` constructor can also take an optional ``max_builds`` parameter which is total number of *different* builders than can run on that worker::
```

Comment:
I think the previous sentence was more precise.
max_builds limit the number of concurent builds that happens on a master. By buildbot design, It happens that one worker can only work on one build of the same builder at the same time. But this is not handled at all by the max_builds logic.

Your sentence imply that you can only configure max_builds on the worker, which is not true.